.github/project.yml

```diff
@@ -1,3 +1,3 @@
 release:
-  current-version: 0.12.0
-  next-version: 1.0.0-SNAPSHOT
+  current-version: 1.1.0
+  next-version: 2.0.0-SNAPSHOT
```

Comment:
We need to increment this also on main branch.

Reply:
I mean, on main:
next-version: 3.0.0-SNAPSHOT

Reply:
I know, this is just here.